### PR TITLE
merge from opensuse

### PR DESCRIPTION
**Pull request by marguerite**
I planned to use `opencc` to convert zh_CN translations for yast/zypper to zh_TW, in order to fill the unmaintained zh_TW branch.

Here're the words I picked when manually comparing the human-translated text.

### CODE REVIEW

File: data/dictionary/TWPhrasesIT.txt [hunk 1]
```diff
@@ -322,3 +322,12 @@ SQL注入攻擊	SQL隱碼攻擊
 默認值	預設值
 點擊	點選
 鼠標	滑鼠
+掛載    裝載
```
Comment (BYVoid) [inline data/dictionary/TWPhrasesIT.txt]: Please use tab instead of 4 spaces.

File: data/dictionary/TWPhrasesIT.txt [hunk 2]
```diff
@@ -322,3 +322,12 @@ SQL注入攻擊	SQL隱碼攻擊
...
+分區    分割區 分割
+分區表    分割區表
+分區程式    磁碟分割程式
+初始化    啟始化
+卷   區塊
```
Comment (BYVoid) [inline data/dictionary/TWPhrasesIT.txt]: Please remove these 4. they are not ONLY IT phrases.

**Comment by tchaikov** [comment]
@marguerite ping?

**Comment by sgalal** [comment]
No update, closing.

**Pull request closed without merging**